### PR TITLE
AF-972: Wrong order of the validation messages

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/AF-972

### CODE REVIEW

File: uberfire-message-console/uberfire-message-console-client/src/main/java/org/guvnor/messageconsole/client/console/MessageConsoleServiceRow.java [hunk 1]
```diff
@@ -16,25 +16,43 @@
...
+    static final Comparator<MessageConsoleServiceRow> DESC_ORDER = (o1, o2) -> {
+        final long sequence1 = o1.sequence;
+        final long sequence2 = o2.sequence;
+        if (sequence1 < sequence2) {
+            return 1;
+        } else if (sequence1 > sequence2) {
+            return -1;
+        }
+        return 0;
+    };
```
Comment (jomarko) [inline uberfire-message-console/uberfire-message-console-client/src/main/java/org/guvnor/messageconsole/client/console/MessageConsoleServiceRow.java]: I believe this anonymous comparator can be written as:
```java
static final Comparator<MessageConsoleServiceRow> NATURAL_ORDER = Comparator.comparingLong(row -> row.sequence);
    
static final Comparator<MessageConsoleServiceRow> DESC_ORDER = NATURAL_ORDER.reversed();
```

Comment (manstis) [reply uberfire-message-console/uberfire-message-console-client/src/main/java/org/guvnor/messageconsole/client/console/MessageConsoleServiceRow.java]: @jomarko I'll give it a try (IDK if GWT supports all of what you propose).

Comment (jomarko) [reply uberfire-message-console/uberfire-message-console-client/src/main/java/org/guvnor/messageconsole/client/console/MessageConsoleServiceRow.java]: Thanks @manstis 

P.S. Dear GWT, please start support all usable stuff :D

**Comment by manstis** [comment]
@jomarko Added new commit changing ```Comparator``` as you proposed... works great!

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
Jenkins please execute full downstream build.

**Pull request merged**